serialization/python/json/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "kiota-serialization-json"
+authors = [{name = "Microsoft", email = "graphtooling+python@microsoft.com"}]
+license = {file = "LICENSE"}
+readme = "README.md"
+classifiers = ["License :: OSI Approved :: MIT License"]
+dynamic = ["version", "description"]
+
+[project.urls]
+Home = "https://github.com/microsoft/kiota#readme"
+
+[tool.flit.module]
+name = "serialization"
+
+[tool.mypy]
+warn_unused_configs = true
+files = "authentication_azure"
```

Comment:
update this?

Reply:
Fixed https://github.com/microsoft/kiota/pull/1186/commits/2e377cfa9d8e9ab4188773594bb020b97e4a2bb4